File: Udemy/Algoritmos e Logica de Programacao/Python/Estrutura Condicional/ProblemaDardo.py
'''Problema "dardo"
No arremesso de dardo, o atleta tem três chances para lançar o dardo à maior distância que conseguir.
Você deve criar um programa para, dadas as medidas das três tentativas de lançamento, informar qual
foi a maior.'''
a:float; b:float; c:float; maior: float;

print("Digite as tres distancias: ")
a = float(input())
b = float(input())
c = float(input())

if a > b and a > c:
    maior = a
elif b > a and b > c:
    maior = b
else:
    maior = c

print(f"MAIOR DISTANCIA = {maior:.2f}")

File: Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaCondicional/src/application/ProblemaDardo.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaDardo {

	public static void main(String[] args) {
		/*
		 * Problema "dardo"
No arremesso de dardo, o atleta tem três chances para lançar o dardo à maior distância que conseguir.
Você deve criar um programa para, dadas as medidas das três tentativas de lançamento, informar qual
foi a maior.
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite as tres distancias: ");
		double a,b,c, maior;
		a = sc.nextDouble();
		b = sc.nextDouble();
		c= sc.nextDouble();
		
		if((a > b) && (a > c)){
			maior = a;
		}else if((b > a)&&(b > c)){
			maior = b;
		}else {
			maior = c;
		}
		
		System.out.println("MAIOR DISTANCIA = "+String.format("%.2f", maior));
				
		
		
		sc.close();

	}

}
